code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -75,8 +75,8 @@
 
 	//Soundy emotey things.
 	var/scream_verb = "screams"
-	var/male_scream_sound		//= 'sound/goonstation/voice/male_scream.ogg' Removed due to licensing, replace!
-	var/female_scream_sound		//= 'sound/goonstation/voice/female_scream.ogg' Removed due to licensing, replace!
+	var/male_scream_sound = list('sound/effects/mob_effects/m_scream_1.ogg','sound/effects/mob_effects/m_scream_2.ogg','sound/effects/mob_effects/m_scream_3.ogg','sound/effects/mob_effects/m_scream_4.ogg','sound/effects/mob_effects/wilhelm_scream.ogg') //CHOMpedit start : Added tgstation screams
```

Comment:
Wilhelm Scream is already used by deathgasp2, btw.

Reply:
We have a death gasp? I have not once seen or heard of it.

Reply:
I think it should be fine if they both exist, considering deathgasp2 only triggers if you manually trigger it yourself, of which I have never seen actually done.

Reply:
> I think it should be fine if they both exist, considering deathgasp2 only triggers if you manually trigger it yourself, of which I have never seen actually done.

It's used plenty on Virgo, and your rand list doesn't include any sorting by rarity, so the wilhem has the same 1/5 chance of being picked as the rest. It's going to quickly go from 'haha easter egg' to really overdone.